docs/getting-started-guides/aws/cloudformation-template.json

```diff
@@ -379,13 +379,13 @@
           "        After=setup-network-environment.service\n\n",
           "        [Service]\n",
           "        EnvironmentFile=/etc/network-environment\n",
-          "        ExecStartPre=/usr/bin/wget -N -P /opt/bin https://storage.googleapis.com/kubernetes-release/release/v0.10.1/bin/linux/amd64/kubelet\n",
+          "        ExecStartPre=/usr/bin/wget -N -P /opt/bin https://storage.googleapis.com/kubernetes-release/release/v0.11.0/bin/linux/amd64/kubelet\n",
           "        ExecStartPre=/usr/bin/chmod +x /opt/bin/kubelet\n",
           "        ExecStart=/opt/bin/kubelet \\\n",
           "        --address=0.0.0.0 \\\n",
           "        --port=10250 \\\n",
           "        --hostname_override=${DEFAULT_IPV4} \\\n",
-          "        --etcd_servers=http://", {"Fn::GetAtt" :["KubernetesMasterInstance" , "PrivateIp"]}, ":4001\\\n",
+          "        --api_servers=", {"Fn::GetAtt" :["KubernetesMasterInstance" , "PrivateIp"]}, ":8080\\\n",
```

Comment:
I'm willing to merge this, but I want to understand why this is working.
On GCE, apiserver only listed on localhost:8080, last I checked.
I'm not sure why this is working.